index.bs

```diff
@@ -2875,7 +2875,7 @@ partial interface MLGraphBuilder {
 };
 </script>
 
-<div>
+<div dfn-for="MLGraphBuilder/gelu(input)" dfn-type="argument">
```

Comment:
@inexorabletash : I notice this one is quoted `dfn-type="argument"`, but none of the others are. e.g.

`<div dfn-for="MLGraphBuilder/hardSigmoid(input, options)" dfn-type=argument>`

Would/could that cause any issues for Bikeshed?

Reply:
TL;DR: no issues

Quoting attributes in HTML is optional if no special characters are involved; my personal rule is if the attribute value is a single word I'll trust it, otherwise use quotes. I don't know if Bikeshed's parser is exactly conformant with HTML's, but for single word values things I trust it.

Reply:
Yeah, it's fine for HTML. Was just wondering about BS. 👌